emcc

```diff
@@ -1491,6 +1491,13 @@ try:
       js_optimizer_queue = []
     js_optimizer_extra_info = {}
 
```

Comment:
you should enable `js_opts`, see https://github.com/kripken/emscripten/blob/master/emcc#L1067

we need to know if any js opt is going to be run later, because it affects what is emitted by the backend.